tests/unit/worker/test_submission_worker.py

```diff
@@ -67,6 +90,72 @@ def setUp(self):
             ),
         )
 
+        self.participant_team = ParticipantTeam.objects.create(
+            team_name="Some Participant Team", created_by=self.user
+        )
+
+        self.challenge_phase = ChallengePhase.objects.create(
+            name="Challenge Phase",
+            description="Description for Challenge Phase",
+            leaderboard_public=False,
+            is_public=True,
+            start_date=timezone.now() - timedelta(days=2),
+            end_date=timezone.now() + timedelta(days=1),
+            challenge=self.challenge,
+            test_annotation=SimpleUploadedFile(
+                "test_sample_file.txt",
+                b"Dummy file content",
+                content_type="text/plain",
+            ),
+            max_submissions_per_day=100,
+            max_submissions_per_month=500,
+            max_submissions=1000,
+            codename="Phase Code Name",
+        )
+
+        self.submission = Submission.objects.create(
+            participant_team=self.participant_team,
+            challenge_phase=self.challenge_phase,
+            created_by=self.challenge_host_team.created_by,
+            status="submitted",
+            input_file=SimpleUploadedFile(
+                "test_sample_file.txt",
+                b"Dummy file content",
+                content_type="text/plain",
+            ),
+            method_name="Test Method",
+            method_description="Test Description",
+            project_url=self.testserver,
+            publication_url=self.testserver,
+            is_public=True,
+            is_flagged=True,
+        )
+
+    def helper_return_submission_input_file_path(self, submission_id, input_file):
```

Comment:
@nikochiko can we rename the method to `get_submission_input_file_path`

Reply:
Yeah, that would be more friendly. Will add the change.